### PR TITLE
added Polish language

### PR DESCRIPTION
**Pull request by majormarcin**

### CODE REVIEW

**Comment by ElPumpo** [comment]
All locales start lower case...

**Comment by majormarcin** [comment]
fixed

File: locales/pl.lua [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+  ['billing'] = 'rachunek',
+  ['invoice_amount'] = 'wartość należności',
+  ['no_players_near'] = 'brak graczy w pobliżu',
+  ['store_veh'] = 'naciśnij ~INPUT_CONTEXT~ aby zdać pojazd',
+  ['drive_search_pass'] = 'jazda w poszukiwaniu ~y~pasażerów',
+  ['customer_found'] = '~g~znalazłeś~s~ klienta, jedź do niego',
+  ['client_unconcious'] = 'twój jest ~r~nieprzytomny xD~w~. Poszukaj innego.',
+  ['arrive_dest'] = '~g~dojechałeś~s~ do celu',
+  ['take_me_to_near'] = '~w~zabierz mnie do~y~ %s~w~, w pobliżu ~y~ %s',
+  ['take_me_to'] = '~w~weź mnie do~y~ %s',
```
Comment (ElPumpo) [inline locales/pl.lua]: Exceptions: strings that start with `~s~`, example:

```lua

-- locale file

['test_1'] = '~o~test~s~',
['test_2'] = '~o~Test~s~',

...

print(_U('test_1')) -- output: 'test'
print(_U('test_2')) -- output: 'Test'

Comment (majormarcin) [reply locales/pl.lua]: do I understand correctly that if the string starts with the example from ~ s ~ then the case after the "closing" of the tag is irrelevant?
Is this something on the principle of automatically changing the letter to a large one when it is needed in the text taken from a language file?

**Comment by ElPumpo** [comment]
👍 merging

**Pull request merged**